### ISSUE TITLE
High CPU, RAM and bandwith usage since upgrade to v2.2.0

### ISSUE THREAD

**Issue by sixeIa** [goharbor/harbor#14461]
**Expected behavior and actual behavior:**
Since v2.2.0 and change in way Harbor manage vulnerabilities (I suppose), launching a global scan seems to impact performances a lot and harbor-core is spammed with things related to vulnerability records conversion.
There is an impact on:
- Harbor server (particularly core which is consuming a lot of CPU/RAM)
- Network
- Database (probably because of a lot if INSERT queries ?)
In case that could be related: database and trivy are "external".
There are 337 images for a size of around 30GB.

**Steps to reproduce the problem:**
Launch a global scan using Interrogation Services tab

**Versions:**
Please specify the versions of following systems.
- harbor version: [2.2.0]

**Additional context:**

Logs:
**First instance**
```
2021-03-16T15:20:52.876188+01:00 harbor-core a9b94fce0017 2021-03-16T14:20:52Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:02.878184+01:00 harbor-core a9b94fce0017 2021-03-16T14:21:02Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:16.728296+01:00 harbor-core a9b94fce0017 2021-03-16T14:21:16Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:29.754247+01:00 harbor-core a9b94fce0017 2021-03-16T14:21:29Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:41.393683+01:00 harbor-core a9b94fce0017 2021-03-16T14:21:41Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:49.874789+01:00 harbor-core a9b94fce0017 2021-03-16T14:21:49Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:02.114344+01:00 harbor-core a9b94fce0017 2021-03-16T14:22:02Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:15.463322+01:00 harbor-core a9b94fce0017 2021-03-16T14:22:15Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:25.333066+01:00 harbor-core a9b94fce0017 2021-03-16T14:22:25Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:39.847823+01:00 harbor-core a9b94fce0017 2021-03-16T14:22:39Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:49.579109+01:00 harbor-core a9b94fce0017 2021-03-16T14:22:49Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:00.891643+01:00 harbor-core a9b94fce0017 2021-03-16T14:23:00Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:12.241496+01:00 harbor-core a9b94fce0017 2021-03-16T14:23:12Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:23.990775+01:00 harbor-core a9b94fce0017 2021-03-16T14:23:23Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:35.950977+01:00 harbor-core a9b94fce0017 2021-03-16T14:23:35Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:47.527937+01:00 harbor-core a9b94fce0017 2021-03-16T14:23:47Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:57.628401+01:00 harbor-core a9b94fce0017 2021-03-16T14:23:57Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
```
**Second one:**
```
2021-03-16T15:17:43.405324+01:00 harbor-core 0622179f775e 2021-03-16T14:17:43Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 997 vulnerability records to the new schema for report ID da156e82-2132-4aa8-aa6f-ab9214490dca and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:17:54.830786+01:00 harbor-core 0622179f775e 2021-03-16T14:17:54Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:18:04.950265+01:00 harbor-core 0622179f775e 2021-03-16T14:18:04Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 6345 vulnerability records to the new schema for report ID b30992dd-5d2f-4d54-b50b-5966acd57c44 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:18:15.950418+01:00 harbor-core 0622179f775e 2021-03-16T14:18:15Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:18:28.247853+01:00 harbor-core 0622179f775e 2021-03-16T14:18:28Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:18:40.583076+01:00 harbor-core 0622179f775e 2021-03-16T14:18:40Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:18:54.970898+01:00 harbor-core 0622179f775e 2021-03-16T14:18:54Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:19:03.907715+01:00 harbor-core 0622179f775e 2021-03-16T14:19:03Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:19:15.816069+01:00 harbor-core 0622179f775e 2021-03-16T14:19:15Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:19:27.387892+01:00 harbor-core 0622179f775e 2021-03-16T14:19:27Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:19:41.228650+01:00 harbor-core 0622179f775e 2021-03-16T14:19:41Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:19:50.654425+01:00 harbor-core 0622179f775e 2021-03-16T14:19:50Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:20:02.107902+01:00 harbor-core 0622179f775e 2021-03-16T14:20:02Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:20:14.042291+01:00 harbor-core 0622179f775e 2021-03-16T14:20:14Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:20:27.487459+01:00 harbor-core 0622179f775e 2021-03-16T14:20:27Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:20:36.834457+01:00 harbor-core 0622179f775e 2021-03-16T14:20:36Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:20:48.554255+01:00 harbor-core 0622179f775e 2021-03-16T14:20:48Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:00.438260+01:00 harbor-core 0622179f775e 2021-03-16T14:21:00Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:12.759234+01:00 harbor-core 0622179f775e 2021-03-16T14:21:12Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:23.857020+01:00 harbor-core 0622179f775e 2021-03-16T14:21:23Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:37.582898+01:00 harbor-core 0622179f775e 2021-03-16T14:21:37Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:48.815401+01:00 harbor-core 0622179f775e 2021-03-16T14:21:48Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:21:59.712336+01:00 harbor-core 0622179f775e 2021-03-16T14:21:59Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:09.882749+01:00 harbor-core 0622179f775e 2021-03-16T14:22:09Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:23.859187+01:00 harbor-core 0622179f775e 2021-03-16T14:22:23Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:36.135115+01:00 harbor-core 0622179f775e 2021-03-16T14:22:36Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:45.201491+01:00 harbor-core 0622179f775e 2021-03-16T14:22:45Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 91cb52f2-b974-44b3-8f9d-7243fa163e2a and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:22:56.561237+01:00 harbor-core 0622179f775e 2021-03-16T14:22:56Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 8e7449aa-87d4-4dd3-ab38-4c9cdb3f7d96 and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:07.834539+01:00 harbor-core 0622179f775e 2021-03-16T14:23:07Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID 0d095979-cb7a-4941-b2b8-fcad8cddc3eb and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:21.813485+01:00 harbor-core 0622179f775e 2021-03-16T14:23:21Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:30.765496+01:00 harbor-core 0622179f775e 2021-03-16T14:23:30Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:44.364088+01:00 harbor-core 0622179f775e 2021-03-16T14:23:44Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:23:53.439472+01:00 harbor-core 0622179f775e 2021-03-16T14:23:53Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
2021-03-16T15:24:03.441135+01:00 harbor-core 0622179f775e 2021-03-16T14:24:03Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 7260 vulnerability records to the new schema for report ID e8f0fbc1-5408-4e29-a327-51ac281f9b5c and scanner Id 27ae0586-80f2-11eb-9e08-9626e59da867
```

**Comment by yanji09** [comment]
take a look to https://github.com/goharbor/harbor/issues/14358

**Comment by sixeIa** [comment]
Hi,
I have looked at this issue but it seems to deal with retention and GC, not trivy and interrogation services.
I used the commands multiple times and it does not affect the strange behavior I've encountered.

**Comment by reasonerjt** [comment]
@sixeIa Could you let us know how many images/artifacts you have in the registry?

**Comment by heww** [comment]
Hi, @sixeIa  I reproduced this issue when there are too many vulnerabilities (eg 7260) in each artifact and scanning with scan all.

I will try to resolve this issue in v2.3.0.

**Comment by gunboe** [comment]
Same issue here, Harbor v2.2.1 installed by Helm chart.
```
# CronJobs:
kubectl logs -f harbor-spi-harbor-jobservice-6657867b88-vf2gm
...
2021-04-27T02:45:38Z [WARNING] [/jobservice/hook/hook_agent.go:121]: Send hook event 'status change: job=d2b5594439d9595b8a67e124, status=Running, check_in=<DATA BLOCK: 5061787 bytes>' to 'http://harbor-spi-harbor-core:80/service/notifications/tasks/269174' failed with error: Post "http://harbor-spi-harbor-core:80/service/notifications/tasks/269174": net/http: timeout awaiting response headers; push hook event to the queue for retrying later
2021-04-27T02:45:48Z [WARNING] [/jobservice/hook/hook_agent.go:121]: Send hook event 'status change: job=f0d4f345f4417e60088685b0, status=Running, check_in=<DATA BLOCK: 5062331 bytes>' to 'http://harbor-spi-harbor-core:80/service/notifications/tasks/269682' failed with error: Post "http://harbor-spi-harbor-core:80/service/notifications/tasks/269682": net/http: timeout awaiting response headers; push hook event to the queue for retrying later
...

# Core:
kubectl logs -f harbor-spi-harbor-core-6d95c6fb47-4t4x6
...
021-04-27T11:48:53Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 5083 vulnerability records to the new schema for report ID 2e4f5e6e-9bad-4caf-a1a5-93f9a68b7dc4 and scanner Id 341e1ea4-9d7c-11eb-9064-1afe79bc7ce9
2021-04-27T11:49:04Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 5227 vulnerability records to the new schema for report ID 08e1e23b-97cc-491f-8d85-211b32e06442 and scanner Id 341e1ea4-9d7c-11eb-9064-1afe79bc7ce9
2021-04-27T11:49:06Z [INFO] [/pkg/scan/postprocessors/report_converters.go:134]: Converted 5337 vulnerability records to the new schema for report ID 70c4164c-b71a-402d-ab89-ae62e245d96f and scanner Id 341e1ea4-9d7c-11eb-9064-1afe79bc7ce9
...
# TOP
kubectl top pods -n harbor
NAME                                               CPU(cores)   MEMORY(bytes)
harbor-spi-harbor-chartmuseum-5f9b4869b-cggtx	   1m           26Mi
harbor-spi-harbor-core-6d95c6fb47-4t4x6            109m         247Mi
harbor-spi-harbor-core-6d95c6fb47-79hg6            1m           46Mi
harbor-spi-harbor-core-6d95c6fb47-jg967            1m           55Mi
harbor-spi-harbor-core-6d95c6fb47-vjxzw            305m         189Mi
harbor-spi-harbor-core-6d95c6fb47-vv246            1m           138Mi
harbor-spi-harbor-core-6d95c6fb47-xxsmq            382m         185Mi
harbor-spi-harbor-exporter-6495dccd4b-bcv9d        1m           9Mi
harbor-spi-harbor-jobservice-6657867b88-gm46g	   24m          59Mi
harbor-spi-harbor-jobservice-6657867b88-gtlkr	   33m          52Mi
harbor-spi-harbor-jobservice-6657867b88-vf2gm	   47m          53Mi
harbor-spi-harbor-notary-server-68ff8959f8-nfhmn   1m           20Mi
harbor-spi-harbor-notary-signer-fdc49b998-cvf4b    0m           9Mi
harbor-spi-harbor-portal-85bf5467ff-t2ffz          1m           14Mi
harbor-spi-harbor-portal-85bf5467ff-tv6kp          1m           9Mi
harbor-spi-harbor-redis-0                          14m          1841Mi
harbor-spi-harbor-registry-6fb5b5b696-2m6r4        1m           70Mi
harbor-spi-harbor-registry-6fb5b5b696-42xfd        1m           80Mi
harbor-spi-harbor-registry-6fb5b5b696-95jjd        1m           75Mi
harbor-spi-harbor-registry-6fb5b5b696-cr456        1m           71Mi
harbor-spi-harbor-registry-6fb5b5b696-llxcr        1m           84Mi
harbor-spi-harbor-trivy-0                          1m           228Mi
```
My DataBase is external and it has 16 vCPU 16GB RAM - constantly 5% CPU and 20 Mbps usage.

**Comment by gunboe** [comment]
Updating, I've got stopped the Loop shutdowning Postgres and Jobservices, but every time I run Trivy Scan the Loop and the heavy DataBase processing come back again.

**Comment by gunboe** [comment]
And yes, I have artifact with more than 5300 vulnerability, like this:
![image](https://user-images.githubusercontent.com/11408669/116492345-68079e80-a872-11eb-83af-df1ba877074a.png)

**Comment by gunboe** [comment]
Hi @heww , there was released out a new version (2.2.2), Was this bug resolved with that?

**Comment by heww** [comment]
Hi @gunboe, Harbor v2.2.2 has some improvements for this problem, but not really resolve this problem.
